plugins/catalog-backend-module-openapi-spec/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@backstage/plugin-catalog-backend-module-openapi-spec",
```

Comment:
Just to think about this name a bit, what do you think makes sense long term for this package? Would we ever add the ability to scan remote OpenAPI endpoints outside of the Backstage ecosystem? It's not particularly needed given the other `openapi` module and it likely makes more sense to add that kind of discovery provider there. With that in mind I'm wondering if something like `backstage-openapi` or `plugins-openapi` might make the naming more accurate?

Reply:
I like `backstage-openapi`, my naming creativity was near 0 on this one. Will update.